keps/sig-release/3344-refine-release-team/README.md

```diff
@@ -0,0 +1,894 @@
+- Wider community members interact with a constantly changing team of people for every release, which makes it harder for them to build working relationships with the release team.
+- With a few notable exceptions, once a contributor ends their time on the release team they don't come back.
+  This "up or out" process makes it hard to do release team work as a long-standing community role.
+
+There are a number of qualities to the current process that are desirable, and should be kept:
+
+- The shadow survey is an excellent "on ramp" into the community, providing contributors with no experience a path to [org membership](https://github.com/kubernetes/community/blob/master/community-membership.md) and it is expected that contributors may only join the release team for a release or two before moving to other parts of the community.
+- The release team having some degree of 'churn' is natural and expected.
+  An entirely static, or effectively static, team is not desirable. 
+
+Balancing these desires, this KEP introduces a long-lived "roster" of release team members.
+This roster is used to assemble release teams instead of a new shadow survey each time.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Continue to provide an "on ramp" for new community members.
+- Provide a more streamlined process for release team shadows, including easier return into future release teams.
+- Provide a "stable" position that a member may continue to be on release teams if they wish.
+- Better handle contributors who wish to take breaks from the team, and then return. 
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Substantively change the makeup and roles of the release team itself.
+- Change how the release team lead, emeritus adviser, and role leads are selected.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### Release Team Roster
+
+We will add to SIG Release a "Release Team Roster".
+The list of roster members will be maintained in a YAML file in https://github.com/kubernetes/sig-release under a new `release-team/roster` directory.
```

Comment:
I'd say we keep to k8s.io and add it as yaml like this: https://github.com/kubernetes/k8s.io/blob/c4b73a3e677f83091a2ab939b289d2f84aad9ff7/groups/sig-release/groups.yaml#L307-L363